projects/ocr.json

```diff
@@ -1,9 +1,12 @@
 {
   "name_cs": "Medical OCR",
   "name_en": "Medical OCR",
+  "name_de": "Medical OCR",
   "subtitle_cs": "Automatické čtení lékařských zpráv",
   "subtitle_en": "Automatic reading of medical reports",
+  "subtitle_de": "Automatisches Lesen von medizinischen Berichten",
   "demo": "https://ocr.stg.mild.blue",
   "desc_cs": "I v jednadvacátém století je hlavním komunikačním nástrojem mezi pacientem a jeho lékařem zpráva vytištěná na kus papíru. Taková zpráva slouží i jako vizitka a představení pacienta v ordinaci jiného lékaře, který se s jejím obsahem chce seznámit. Jakmile je lékařských zpráv více, stává se takové seznámení se s pacientovým zdravotním stavem pro lékaře velmi časově náročným a exponenciálně se zvyšuje šance, že lékař nějakou důležitou informaci přehlédne. Cílem projektu je vytvořit nástroj, který vytištěnou lékařskou zprávu nejen opět zdigitalizuje, ale také analyzuje a předpřipraví pro další použití. Prototyp je ve fázi integrace do aplikace zScanner.",
-  "desc_en": "Even in the 21st century many things are kept on paper. This includes medical reports. Not only Czech doctors spend hours of their precious time studying reports from other hospitals and rewriting those into new medical reports they are writing. Our tools lets the doctor digitize the report easily, analyze and pre-process it and then use the information from the report elsewhere. Currently we are integrating the tool into an opensource medical app zScanner."
+  "desc_en": "Even in the 21st century many things are kept on paper. This includes medical reports. Not only Czech doctors spend hours of their precious time studying reports from other hospitals and rewriting those into new medical reports they are writing. Our tools lets the doctor digitize the report easily, analyze and pre-process it and then use the information from the report elsewhere. Currently we are integrating the tool into an opensource medical app zScanner.",
+  "desc_de": "Auch im 21. Jahrhundert werden viele Dinge auf Papier gehalten. Medizinische Berichte stellen keine Ausnahme dar. Nicht nur die tschechischen Ärzte verbringen Stunden ihrer wertvollen Zeit mit den Berichten. Sie müssen die Berichte aus anderen Krankenhäusern studieren und dann neue selbst schreiben. Mit unseren Tools kann der Arzt den Bericht einfach digitalisieren, analysieren und vorverarbeiten und die Informationen aus einem Bericht in anderem verwenden. Derzeit integrieren wir das Tool in eine OpenSource Medical App zScanner."
```

Comment:
- viele Dinge auf Papier. I v te anglictine je to na zvazeni, ale v nemcine to podle zni vylozene hloupe. Zkusite to preformulovat prosim?
- ta anglicka verze je zde, koukam, dost slaba. Mohl byste to vice napasovat na tu ceskou variantu?

Reply:
Přepsáno na novou verzi. Je to takto v pořádku?